FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Mutual_Auth_wolfSSL/README_wolfSSL.md

```diff
@@ -10,18 +10,112 @@ By double-clicking the solution file named "**mqtt_mutual_auth_demo_wolfSSL.sln*
 
 All required settings for wolfSSL have been set in the user_settings.h header file included in the RTOSDemo folder in the solution explorer pane. For this demo to work, you need to set the following information:
 
-1. choose interface to use
-2. set MQTT broker endpoint URL
-2. set root CA certificate file(.pem) path
-3. set private key file(.pem) path
+1. set broker endpoint
+2. set root CA certificate
+3. set client certificate
+4. set private key
+5. choose interface to use
  
+If even one of the above 1 to 4 is not set, an error will occur at build time. You should open **demo_config.h** to set them.
 
+<br>
 
-# Choose an interface to use
-You should choose an interface to use to configure the demo. However you may not know how to choose it. The demo will give you  good guidance. Follow the steps below.
+# Set Broker endpoint
 
-1. Build the RTOSDemo project
-2. Run the RTOSDemo.exe 
+A broker endpoint is a url that represents where MQTT subscribers and publishers access. In case your device is going to access AWS IoT device data endpoints, the endpoint would be the following format: ***account-specific-prefix*.iot.*aws-region*.amazonasw.com**.

```

Comment:
nit: amazonasw -> amazonaws.

A useful command to look this up could be `aws iot describe-endpoint --endpoint-type iot:Data-ATS`

Reply:
Fixed it.